packages/api/helper/googleSheetsAPIConnection.js

```diff
@@ -0,0 +1,31 @@
+const { google } = require('googleapis');
+
+const client = new google.auth.JWT(
+  process.env.GOOGLE_SHEETS_CLIENT_EMAIL,
+  null,
+  process.env.GOOGLE_SHEETS_PRIVATE_KEY,
+  ['https://www.googleapis.com/auth/spreadsheets']
+);
+
+client.authorize(function (err, tokens) {
+  if (err) {
+    console.log(err);
+    return;
+  } else {
+    console.log('Connected!');
+    gsrun(client);
+  }
+});
+
+async function gsrun(cl) {
+  const gsapi = google.sheets({ version: 'v4', auth: cl });
+
+  const opt = {
+    spreadsheetId: '1GGtT9THPQZBHJSFcQHNu1mYBOdRLeBHZjTz0B5gcY2U',
+    range: 'Sheet1',
+  };
+
+  let data = await gsapi.spreadsheets.values.get(opt);
+
+  console.log(data.data.values);
+}
```

Comment:
This doesn't have any exports, so I'm not sure what kind of feedback to provide about the functionality. Do we know what we'd like this helper to return?

In other words, do we want to return the data from this helper? Or do we want to return a function that can repeatably fetch the data? Or something else?